cutevariant/gui/columnquerywidget.py

```diff
@@ -16,13 +16,11 @@ def __init__(self):
 
     @property
```

Comment:
using virtual method as a property, it forces you to add extra keywords "@property" for all widget based on QueryPluginWidget.  It may be discouraging. 
I suggest to keep property for setter/getter query only. And create one virtual methods "query_changed()" to overload. 
What do you think ?

Reply:
Indeed, these decorators have more meaning in classes that carry the attribute directly in private rather than in those that initialize the attribute of other objects (models for example).
But this is an operation that a setter does.

Calling a function automatically after an assignment may seem curious, but in fact this is also the purpose of a getter/setter.

So I do not know what else to say, I think the two methods are the same, the properties are just cleaner when you look at the api of a class and we avoid heavy parentheses.
But yes, it can be quite confusing in the middle of all Qt objects that do not use these possibilities of the Python language.